docs/best_practices/python.md

```diff
@@ -0,0 +1,167 @@
+# Python best practices
+
+## Import
+
+### Avoid `from module_foo import symbol_bar`
+
+In general, it is a good practice to avoid the form `from foo import bar`
+because it introduces two distinct bindings (`bar` is distinct from `foo.bar`)
+and when the binding in one namespace changes, the binding in the other will
+not…
```

Comment:
Did we?

Personally, I find it a bit sad that we still limit ourselves to ASCII in 2019 (come on, Unicode is here since 1991, and UTF-8 since 1996: they are hardly bleeding edge technologies), and using Unicode tends to show early if a tool sucks or is badly configured.

Maybe we need a "How to write Markdown" best practice guide as well :D

Reply:
Ok, I thought a discussion with Nicolas lead to this decision. Personally, I'm fine with using Unicode.